src/main/java/no/fint/betaling/service/FetchClaims.java

```diff
@@ -0,0 +1,58 @@
+package no.fint.betaling.service;
+
+import no.fint.betaling.model.Claim;
+import no.fint.betaling.model.ClaimStatus;
+import no.fint.betaling.model.ClaimsDatePeriod;
+import no.fint.betaling.repository.ClaimRepository;
+
+import java.util.List;
+
+import static no.fint.betaling.service.ClaimService.claimsDatePeriodToTimestamp;
+
+public class FetchClaims {
```

Comment:
Jeg er ikke overbegeistret for dette navnet. Et rent verb er sjeldent et godt klassenavn. Typisk bruker vi også prototypene til Spring: Controller, Service, Repository. 
F.eks. klinger ClaimFetcherService eller ClaimStatusSelectorService bedre i mine øre :)